gammapy/maps/hpx.py

```diff
@@ -1731,6 +1732,23 @@ def solid_angle(self):
         import healpy as hp
         return Quantity(hp.nside2pixarea(self.nside), 'sr')
 
+    def get_region_idx(self, region):
+        """Return idx of pixels inside region
```

Comment:
I don't think this description is correct. It returns a boolean mask, not a pixel idx array, no?